how-to-docs/rds-provisioning.md

```diff
@@ -0,0 +1,137 @@
+# Provisioning an AWS RDS database into the Platform
+
+This requires docker and AWS credentials for hod-dsp-[env] to create the resources and dsp-ci for drone to deploy to the platform. In this example env is _dev_ and the service_name is _myexample_
+
+
+### Workflow
+ - Pull stacks-hod-platform repo
+ - Configure templates in feature branch and push
+ - Create db password and upload secrets
+ - Peer review then merge to master, optionally run script to deploy to prod
```

Comment:
You actually need to create the password before pushing or the validation will fail